### PR TITLE
[1.13.x-blue][KOGITO-7457] - Rebranding changes on the kogito-operato…

### PR DESCRIPTION
**Pull request by davidesalerno**
…r (fixing package into Makefile)

See: [KOGITO-7457](https://issues.redhat.com/browse/KOGITO-7457)
 
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've ran `make before-pr` and everything is working accordingly
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [x] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Comment by radtriste** [comment]
I guess you should update the `config` folder and regenerate the bundle from there

**Comment by davidesalerno** [comment]
> I guess you should update the `config` folder and regenerate the bundle from there

I did it but it seems that the manifests generated by the operator-sdk using kustomize are ignoring the ./config/manifests/rhpam/bases/ibm-bamoe-kogito-operator.clusterserviceversion.yaml file

**Review by radtriste — APPROVED** [review]
good to me as long as the Jenkins RHPAM test is green

File: Makefile.rhpam [hunk 1]
```diff
@@ -26,53 +26,53 @@ all: generate manifests container-build
...
-	$(CONTROLLER_GEN) rbac:roleName=manager-role paths="./controllers/rhpam" output:rbac:artifacts:config=config/rbac/rhpam
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./apis/rhpam/..." output:crd:artifacts:config=config/crd/bamoe/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role paths="./controllers/rhpam" output:rbac:artifacts:config=config/rbac/bamoe
 	$(CONTROLLER_GEN) webhook paths="./..."
 	./hack/kogito-module-api.sh --enable
 
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	echo "calling IBM BAMOE generate......................................"
 	./hack/kogito-module-api.sh --disable
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths=./...
```
Comment (spolti) [inline Makefile.rhpam:37]: I think we shouldn't remove it.

Comment (spolti) [inline Makefile.rhpam:37]: is this intended to be removed?

**Comment by spolti** [comment]
cvp is green.

**Pull request merged**